_posts/2020-01-17-event-store-6.0.0-preview-2-release.md

```diff
@@ -0,0 +1,146 @@
+
+With these changes, the projections should have an average of 10 minutes to recover before faulting.
+
+## Changes to projections
+
+In our ongoing effort to improve projections we have made a number of changes.
+
+Some projection selectors and options were removed:
+
+- `fromStreamCatalog` selector
+- `fromStreamsMatching` selector
+- `disableParallelism` option
+- `catalogTransform` option
+
+These selectors were only usable in queries or transient projections. If you were using them, you need to recreate any affected queries or transient projections.
+
+As only `fromStreamsMatching` was officially documented, it was the only one of these officially supported.
+
+If you were making use of `fromStreamsMatching`, you can replace it with `fromAll` and an appropriate where clause.
+
+## Disable clone nodes by default
+
+Following the release of Read-Only Replicas, Clones have been disabled by default.
+
+The default behaviour now is for excess nodes to be terminated when they join the cluster.
+
+If you still want to make use of Clones, you can re-enable them by setting the `--unsafe-allow-surplus-nodes` option on all nodes in the cluster.
+
+## Fix race condition in TCP connections
+
+We addressed an issue whereby data could be dispatched even after the tcp connection has been closed.
+
+Improvements to Event Store gRPC protocol buffer design (2190)
+
+> Note: this does not affect users of Event Store, only those implementing client SDKs.
+
+We have addressed an issue whereby some code generators for languages (notably Rust and Scala) were failing because of the use of the plural form of options.
+
+> Note: if you are developing a gRPC-based client, please update your proto contracts.
+
+## Return structured vs string based UUIDs from gRPC reads
+
+> Note: this does not affect users of Event Store, only those implementing client SDKs.
+
+We have received some great feedback from the community around gRPC, more notably community members who have started building out gRPC clients.
+
+One of the pain points was that it was difficult to interpret the structured UUID (most and least significant bits) and that we should provide the ability to indicate what the format of the UUID should be returned as when performing reads.
+
+With the change, the client has the ability to specify whether or not they want a structured or string based version of the UUID.
+
+> Note: if you are developing a gRPC client, please update your proto contracts.
+
+## Use event counters for performance statistics
+
+With our move to .NET Core, the performance counters that we use will only work on Windows and therefore we have made use of the `Microsoft.Diagnostics.NETCore.Client` package to leverage the new tracing capabilities in .NET Core to surface some of the statistics in our /stats endpoint.
+
+Currently system-wide CPU and memory usage are not exposed via the statistics endpoint on Linux or macOS.
```

Comment:
```suggestion
> Note: Currently system-wide CPU and memory usage are not exposed via the statistics endpoint on Linux or macOS. This will be rectified in the release candidate.
```